core/tasks.py

```diff
@@ -17,9 +17,6 @@
 from viur.core.config import conf
 from viur.core.utils import currentLanguage, currentRequest, currentSession
 
-logger = logging.getLogger("viur.core.tasks")
```

Comment:
Let's fix this in a separate pull request for specific modules. I need a working logging for this PR.